requirements.txt

```diff
@@ -14,6 +14,7 @@ optree>=0.13.0
 packaging
 psutil
 pyyaml
+pyzstd
```

Comment:
Why add it here?

Reply:
I've removed it for now.  If this causes issues later, we can revisit.